### ISSUE TITLE
When file picker is open and window is resized the main thread panics

### ISSUE THREAD

**Issue by wojciechkepka** [helix-editor/helix#17]
As the title says, when the file picker window is open and I split the terminal window verticaly using tmux there is a panic:
```
thread 'main' panicked at 'index out of bounds: the len is 3780 but the index is 3780', helix-tui/src/buffer.rs:185:14
```

When I first split the window and then open the file picker it correctly scales later on when the split is closed and opened again.

**Comment by akgvn** [comment]
I get the same error when trying the launch `hx` on Windows (I tried using cmd, powershell, cmder and Windows Terminal):
```
thread 'main' panicked at 'index out of bounds: the len is 6355 but the index is 6355', helix-tui\src\buffer.rs:185:14
```

**Comment by archseer** [comment]
I think it should be fixed by c0332bd935bb7c016278d43a526e588558a0cab1

**Comment by archseer** [comment]
Fixed by c0332bd935bb7c016278d43a526e588558a0cab1
